server/start.go

```diff
@@ -340,6 +344,11 @@ func startInProcess(ctx *server.Context, clientCtx client.Context, appCreator ty
 		app.RegisterTendermintService(clientCtx)
 	}
 
+	if ctx.Viper.GetBool(srvflags.JSONRPCEnableMetrics) {
```

Comment:
This should probably be in the config just as you did with the MetricsAddress

Reply:
If this is specified in the config, a user can enable it from config but not pass the `--metrics` flag resulting in metrics not being enabled. This will cause confusion for user. Resolving this conversation